src/lib/logical_query_plan/aggregate_node.hpp

```diff
@@ -49,21 +50,22 @@ class AggregateNode : public AbstractLQPNode {
    *
    * NOTE: These functions will possibly result in a full recursive traversal of the ancestors of this node.
    */
-  std::optional<ColumnID> find_column_id_for_expression(const std::shared_ptr<Expression>& expression) const;
-  ColumnID get_column_id_for_expression(const std::shared_ptr<Expression>& expression) const;
+  std::optional<LQPColumnReference> find_column_reference_by_expression(
```

Comment:
`find_column_reference`?

Reply:
or just `find_column`? Need to test whether this hides the name in the base class

Reply:
seems like it does (https://stackoverflow.com/questions/32946364/c-inheritance-and-name-hiding), will go for `find_column_by_expression`

Reply:
ok